apps/speed.c

```diff
@@ -933,11 +1459,28 @@ int speed_main(int argc, char **argv)
         goto end;
     }
 
+    if (async_jobs > 0) {
+        if (!ASYNC_init(1, 0, 0)) {
```

Comment:
Would it be legal your opinion to initialize the pool without setting an explicit `max_size`?
For example:

```
ASYNC_init(1, 0, async_jobs)
```

This code in async.c explicitly prevent that:

```
   if (init_size > max_size) {
        ASYNCerr(ASYNC_F_ASYNC_INIT_THREAD, ASYNC_R_INVALID_POOL_SIZE);
        return 0;
    }
```